packages/wp-story-editor/src/components/documentPane/publish/publishTime.js

```diff
@@ -102,7 +102,7 @@ function PublishTime() {
 
   // Floating date means an unset date so that the story publish date will match the time it will get published.
   const floatingDate =
-    ['draft', 'pending', 'auto-draft'].includes(status) &&
+    ['draft', 'pending', 'auto-draft', 'future'].includes(status) &&
     (date === modified || date === null);
```

Comment:
Is this condition ever true for scheduled posts? I think a post with status `future` will always have a non-null date, and the scheduled date will always be different from the modified date.
In other words, do we need this? Can we revert this change?

Reply:
A story will be marked as `future` whenever we schedule it. Thus, including it solved the issue!

Updating `floatingDate` will [display `"Immediate"` label](https://github.com/GoogleForCreators/web-stories-wp/blob/f50d7e2594692bd1e0e9f61ade540386b8ccdcba/packages/wp-story-editor/src/components/documentPane/publish/publishTime.js#L112) and [show `Reset` button](https://github.com/GoogleForCreators/web-stories-wp/blob/f50d7e2594692bd1e0e9f61ade540386b8ccdcba/packages/wp-story-editor/src/components/documentPane/publish/publishTime.js#L153-L156).

Reply:
Well, I guess if it works it works! :-)